src/html5_parser/parser/mod.rs

```diff
@@ -2254,7 +2257,7 @@ impl<'a> Html5Parser<'a> {
                         // ignore token
                         return;
                     }
-                    let node_id = node_id.unwrap();
+                    let node_id = node_id.expect("node_id");
```

Comment:
Convention: use `expect(...)` when we know from earlier code that a panic shouldn't happen.